SmartTutorTests/Integration/LearnerTests.cs

```diff
@@ -23,11 +24,12 @@ public void Successfully_logins()
         {
             using var scope = _factory.Services.CreateScope();
             var controller = new LearnerController(_factory.Services.GetRequiredService<IMapper>(),
-                scope.ServiceProvider.GetRequiredService<ILearnerService>());
+                scope.ServiceProvider.GetRequiredService<ILearnerService>(),
+                scope.ServiceProvider.GetRequiredService<IKeycloakService>());
```

Comment:
An example where we can inject the "NoAuthProvider" instead of the "KeycloakAuthProvider"